### DISCUSSION TITLE
[Enhancement] [select] Select和Select V2添加支持多选tag点击的功能（并且不影响原有功能）

### DISCUSSION THREAD

**Discussion by wizardAEI** [element-plus/element-plus#10000] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `select`

## Description

  选项框多选时，对于某个tag，它作为一个label可能会附带一些其他的次要信息，我的某个业务中希望用户点击标签时可以从某些地方显示出来这些附加信息，这就需要点击标签时可以得到它的key（但是当我不需要这个点击事件时，组件的交互应该和原来一样，即点击tag展开select）。
  这个功能的Select和Select V2组件以及他们的测试样例和文档我已经在dev分支上提了[PR](https://github.com/element-plus/element-plus/pull/10001)，还请官方大大们看一下这个功能是否需要。

### Additional comments

_(empty)_

**Comment by jinkunaier** [comment]
支持插槽渲染也可以做到，这些都是基操，其它一些UI框架上都有，我也提了！顶起来！！
